src/snapshot.zig

```diff
@@ -312,6 +340,35 @@ fn processSnapshotFile(snapshot_path: []const u8, gpa: Allocator) !bool {
     try file.writer().writeAll("\n");
     try file.writer().writeAll(parse_buffer.items);
 
+    // If flag --fuzz-corpus is passed, so write the SOURCE to our corpus
+    if (maybe_fuzz_corpus_path != null) {
+
+        // create a pseudo-random name for our file
+        const rand_file_name = [_][]const u8{
+            maybe_fuzz_corpus_path.?, &[_]u8{
+                rand.intRangeAtMost(u8, 'a', 'z'),
+                rand.intRangeAtMost(u8, 'a', 'z'),
+                rand.intRangeAtMost(u8, 'a', 'z'),
+                rand.intRangeAtMost(u8, 'a', 'z'),
+                rand.intRangeAtMost(u8, 'a', 'z'),
+                rand.intRangeAtMost(u8, 'a', 'z'),
+                rand.intRangeAtMost(u8, 'a', 'z'),
+                rand.intRangeAtMost(u8, 'a', 'z'),
+            },
+        };
```

Comment:
Maybe these files should end with `.roc`?

Reply:
I'll add this in my follow up PR #7669

Reply:
Also asked about this in [zulip](https://roc.zulipchat.com/#narrow/channel/395097-compiler-development/topic/zig.20compiler.20-.20fuzzing/near/503905890)